src/pages/Fares.tsx

```
@@ -0,0 +1,136 @@
+                                        Tickets: {journey.tickets.map(ticket => (
+                                            <li key = { ticket.fareId } className = "ticket-item">
+                                                <p>{ticket.name}</p>
+                                                <p>Price: {ticket.priceInPennies / 100}£</p>
+                                            </li>
+                                        ))}
+                                    </ul>
+                                </li>
+                            ))
+                        }
+                    </ol>
+                </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)